ultimarc/devices/_mappings.py

```diff
@@ -6,6 +6,23 @@
 # Mappings for devices.
 #
 
+#
+# PAC 2015 or newer debounce values
+#
+IPACSeriesDebounce = {
+    'standard': 0,
+    'none': 0x01,
+    'short': 0x02,
+    'long': 0x03
+}
+
+
+def get_ipac_series_debounce_key(val):
+    for key, value in IPACSeriesDebounce.items():
+        if val == value:
+            return key
+    return None
```

Comment:
You can simplify this by change code to:
`if key not in IPACSeriesDebounce:
     return val
return None
`

Reply:
I am looking to return the key not the value.  I think I will keep it the way I have it after adding the logging in based on your comments in mini_pac.py file.